src/config/external-applications.md

```diff
@@ -0,0 +1,83 @@
+# External Applications
+
+## Programming Languages
+
+> The Void repositories have a number of Python and Lua packages. If possible,
+> install packages from the Void repositories or consider packaging the library
+> or application you need. Packaging your application allows for easier system
+> maintenance and can benefit other Void Linux users, so consider making a pull
+> request for it. The contribution instructions can be found
+> [here](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md).
```

Comment:
Further to [this recent comment of mine](https://github.com/void-linux/void-docs/pull/217#discussion_r419204420), i think this text shouldn't be quoted, particularly if the intent of the quoting is to achieve layout effects - i'd prefer to keep our markup as semantic as possible.